### DISCUSSION TITLE
Ubuntu virtual machine error

### DISCUSSION THREAD

**Discussion by Ed-die-lira** [Cyfrin/foundry-full-course-cu#190] [Q&A] [answered]
WslRegisterDistribution failed with error: 0x80370102
Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS.

**Comment by ChefAharoni** [comment]
Hi @Ed-die-lira, please provide more details to your question. 
What are you trying to do?
Are you running your Ubuntu machine as a VM on your computer?
What is your Ubuntu version?

Giving more details can help others help you better :)

**Reply by Ed-die-lira** [reply]
Hi, i m try install  a ubuntu VM in my windows 10, like the first video of Patrick teaching in Lesson 6, but after reboot of PC, show this message.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/135538064/c350d69f-ba4e-4366-bd22-6cefde936024)

**Reply by ChefAharoni** [reply]
Have you enabled the necessary requirements shown in the message?

You can view the steps to solve it [here](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed)

**Reply by Ed-die-lira** [reply]
i follow the path, and activated the requirements and show this message.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/135538064/74c84235-9374-4f4e-96fd-1fd64630b33b)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/135538064/7660af14-1ece-405b-a0d1-04cfd486e929)

**Reply by ChefAharoni** [reply]
`<vname>` should be **your** virtual machine's name, it's a place holder they put in the instructions for you to fill.

**Reply by Ed-die-lira** [reply]
I managed to solve thanks, there a option in the site which i don't seen before, i needed change the option in bios, i was thinking what was activated before. thanks for your help.
https://www.bleepingcomputer.com/tutorials/how-to-enable-cpu-virtualization-in-your-computer-bios/

**Reply by ChefAharoni** [reply, marked as answer]
Great, I'm glad to hear.
Please mark this discussion as answered so other people will know the issue is resolved :)
